flink-connector-opensearch/src/test/java/org/apache/flink/connector/opensearch/sink/OpensearchWriterITCase.java

```diff
@@ -174,7 +175,8 @@ void testIncrementByteOutMetric() throws Exception {
                 new BulkProcessorConfig(flushAfterNActions, -1, -1, FlushBackoffType.NONE, 0, 0);
 
         try (final OpensearchWriter<Tuple2<Integer, String>> writer =
-                createWriter(index, false, bulkProcessorConfig, metricGroup)) {
+                createWriter(
```

Comment:
You probably don't need this change since `createWriter` has an overloaded version without failure handler

Reply:
Agreed, it will be better not to touch that line. I did it because it needed to pass the metricGroup.
So I am fixing the situation by adding yet another constructor, with metricGroup but without failureHandler.

Reply:
No need for constructor, `createWriter` is a method in this class